src/rust/ide/lib/view/src/graph_editor/builtin/visualization/native/documentation.rs

```diff
@@ -0,0 +1,184 @@
+        dom.dom().set_style_or_warn("pointer-events"  ,"auto"                          ,&logger);
+        dom.dom().set_style_or_warn("border-radius"   ,format!("{}px", CORNER_RADIUS)  ,&logger);
+        dom.dom().set_style_or_warn("width"           ,format!("{}px", DOC_VIEW_WIDTH) ,&logger);
+        dom.dom().set_style_or_warn("height"          ,format!("{}px", doc_view_height),&logger);
+
+        scene.dom.layers.main.manage(&dom);
+        ViewModel {dom,logger,size}.init()
+    }
+
+    fn init(self) -> Self {
+        self.reload_style();
+        self
+    }
+
+    /// Sets size of the documentation view.
+    fn set_size(&self, size:Vector2) {
+        self.size.set(size);
+        self.reload_style();
+    }
+
+    /// Generates HTML documentation for documented suggestion.
+    fn gen_doc(doc: String) -> FallibleResult<String> {
+        let parser = parser::DocParser::new()?;
+        let output = parser.generate_html_doc_pure(doc);
+        Ok(output?)
+    }
+
+    fn receive_data(&self, data:&Data) -> Result<(),DataError> {
+        let data_inner = match data {
+            Data::Json {content} => content,
+            _                    => todo!(),
+        };
+
+        let data_str = serde_json::to_string_pretty(&**data_inner);
+        let data_str = data_str.unwrap_or_else(|e| format!("<Cannot render data: {}>", e));
+        // Fixes a Doc Parser Bug - to be removed when rewritten to rust
+        let data_str = data_str.replace("\\n", "\n");
+        let data_str = data_str.replace("\"", "");
+
+        let output = ViewModel::gen_doc(data_str);
+        let output = output.unwrap_or_else(|_| String::from(PLACEHOLDER_STR));
+        // Fixes a Doc Parser related idea, where stylesheet was a separate file
+        let output = output.replace(r#"<link rel="stylesheet" href="style.css" />"#, "");
+
+        let data_str = format!(r#"<div class="docVis">{}{}</div>"#, get_doc_style(), output);
+        self.dom.dom().set_inner_html(&data_str);
+        Ok(())
+    }
+
+    /// Loads an HTML file into the documentation view when there is no docstring available.
+    fn load_no_doc_screen(&self) {
+        let data_str = format!(r#"<div class="docVis">{}{}</div>"#, get_doc_style(), PLACEHOLDER_STR);
+        self.dom.dom().set_inner_html(&data_str)
+    }
+
+    fn reload_style(&self) {
+        self.dom.set_size(self.size.get());
+    }
+}
+
+// ============
+// === View ===
+// ============
+
+/// Visualization that renders the given documentation as a HTML page
+#[derive(Clone,CloneRef,Debug,Shrinkwrap)]
+#[allow(missing_docs)]
+pub struct View {
+    #[shrinkwrap(main_field)]
+    pub model : ViewModel,
+    pub frp   : visualization::instance::Frp,
+    network   : frp::Network,
+}
+
+impl View {
+    /// Definition of this visualization.
+    pub fn definition() -> Definition {
+        let path = Path::builtin("Documentation View Visualization (native)");
```

Comment:
remove "Visualization" word. Its not needed. Remove also "(native)". It was used in examples, but here its not needed. USer do not need to know how it was implemented.